OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java

```diff
@@ -1569,6 +1595,25 @@ public boolean deleteWptPt(WptPt point) {
 
 			return points.remove(point);
 		}
+		
+		public boolean deleteWptPt(String wptName) {
```

Comment:
class is overloaded with lines of code, refactoring is needed and this method is not a good example that should be here especially wptName is not unique key!

Reply:
wptName is unique key for group